aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -249,6 +249,11 @@ Tensor& randperm_out(Tensor& result, int64_t n, Generator* generator) {
   return result;
 }
 
+Tensor randint(const Type& dtype, IntList size, Generator* generator) {
+  Tensor result = dtype.tensor(size);
+  return result.random_(0, 1, generator);
```

Comment:
`randint` should take a range....

Reply:
Oops! commited wrong file, so sorry, my bad!! Will update and commit.